support/build/_docker/cedar-14.Dockerfile

```diff
@@ -3,9 +3,10 @@ FROM heroku/cedar:14
 WORKDIR /app
 ENV WORKSPACE_DIR=/app/support/build
 
-RUN apt-get update
-RUN apt-get install -y python-pip
+RUN apt-get update && apt-get install -y python-pip
 
-RUN pip install 'bob-builder>=0.0.10' 's3cmd>=1.6.0'
+COPY requirements.txt /app/requirements.txt
 
-ADD . /app
+RUN pip install -r /app/requirements.txt
+
+COPY . /app
```

Comment:
@dmathieu: "For other items (files, directories) that do not require ADD’s tar auto-extraction capability, you should always use COPY.", see https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/add-or-copy